src/Components/TicTacToe/TicTacToeView.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import styles from "./TicTacToe.module.css";
+
+function Square({value, onClick}) {
+  const squareStyle = value === "X" ? styles["square-cross"] : styles["square-circle"];
```

Comment:
Although it's not needed at the moment, there should probably be a separate style for empty squares.

Reply:
Thanks for the feedback! I now also check for ties :)